src/vs/platform/diagnostics/node/diagnosticsService.ts

```diff
@@ -61,12 +66,27 @@ export async function collectWorkspaceStats(folder: string, filter: string[]): P
 		{ tag: 'dockerfile', filePattern: /^(dockerfile|docker\-compose\.ya?ml)$/i }
 	];
 
+	const rootPathPatterns: RootFilePatterns[] = [
```

Comment:
Uhm I am worried these regex may create a bunch of false positives, especially more generically-named ones like `box`: they would be triggered for any folder called `box`. Instead, it should be `~/Box` on macOS and `%HOMEPATH%/Box` on Windows.